sdl_android/src/main/java/com/smartdevicelink/security/SdlSecurityBase.java

```diff
@@ -84,12 +86,22 @@ public void setAppId(String val) {
     	appId = val;
     }
 
+    @Deprecated
     public static Service getAppService() {
-    	return appService;
+        return appService;
     }
-    
+
+    @Deprecated
     public static void setAppService(Service val) {
-    	appService = val;
+        appService = val;

```

Comment:
I would also check if `appService != null` and if true, set `context` from the service's context

Reply:
added it